diff --git a/node_modules/expo-audio/ios/AudioRecordingRequester.swift b/node_modules/expo-audio/ios/AudioRecordingRequester.swift
index eac41f3..96492a1 100644
--- a/node_modules/expo-audio/ios/AudioRecordingRequester.swift
+++ b/node_modules/expo-audio/ios/AudioRecordingRequester.swift
@@ -12,10 +12,10 @@ public class AudioRecordingRequester: NSObject, EXPermissionsRequester {
     var status: EXPermissionStatus
 
     guard (Bundle.main.infoDictionary?["NSMicrophoneUsageDescription"]) != nil else {
-      EXFatal(EXErrorWithMessage("""
+      assertionFailure("""
         This app is missing NSMicrophoneUsageDescription, so audio services will fail.
         Add one of these keys to your bundle's Info.plist.
-      """))
+      """)
       return ["status": EXPermissionStatusDenied]
     }
 
